pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -72,18 +72,23 @@ type Controller struct {
 	networkPolicyWatcher  *watcher
 	appliedToGroupWatcher *watcher
 	addressGroupWatcher   *watcher
+	restartFullSyncGroup  *sync.WaitGroup
+	restartSynced         bool
 }
 
 // NewNetworkPolicyController returns a new *Controller.
 func NewNetworkPolicyController(antreaClientGetter agent.AntreaClientProvider,
 	ofClient openflow.Client,
 	ifaceStore interfacestore.InterfaceStore,
 	nodeName string,
+	restartFullSyncGroup *sync.WaitGroup,
```

Comment:
As discussed before, the wait group to sort batch update and starting workers can be controller internal? No need to initialize it outside and pass in.

Reply:
Done